.github/workflows/create-package.yml

```diff
@@ -0,0 +1,40 @@
+﻿name: Create and publish package
+
+on:
+  release:
+    types:
+      - created
+
+jobs:
+  package-release:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        configuration: [BelowZero, Subnautica]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Get artifact from build
+      uses: dawidd6/action-download-artifact@v2
+      with:
+        workflow: build.yml
+        name: SCHIZO-${{ matrix.configuration }}.dll
+        path: Install/${{ matrix.configuration }}/BepInEx/plugins/SCHIZO/
+
+    - name: Create release archive
+      run: |
+          cd Install/${{ matrix.configuration }}
+          zip -r9 ../mod-and-deps-${{ matrix.configuration }}.zip *
+          cd BepInEx/plugins
+          zip -r9 ../../../mod-only-${{ matrix.configuration }}.zip SCHIZO
+          cd ../../../Common
+          zip -r9 ../mod-and-deps-${{ matrix.configuration }}.zip *

```

Comment:
```suggestion
          cd Install
          zip -r9 SCHIZO-${{ matrix.configuration }}-with-dependencies.zip Common/*
          zip -r9 SCHIZO-${{ matrix.configuration }}-with-dependencies.zip ${{ matrix.configuration }}/*
          cp ${{ matrix.configuration }}/BepInEx/plugins/SCHIZO/SCHIZO.dll ./SCHIZO-${{ matrix.configuration }}.dll
```

Reply:
Doing it this way will create this folder structure within the zip file:
- `Common/BepInEx/...`
- `BelowZero/BepInEx/...`

The BepInEx folder won't be merged.

Don't think there's a flag to selectively strip off the prefix

Reply:
The best way to do this would be to use a series of `pushd` and `popd`, but I figured that might be overkill here